mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php

```diff
@@ -220,27 +226,13 @@ private function getEmailVolumeLimitReachedMessage(): string {
     return "{$message}<br/>";
   }
 
-  private function getPendingApprovalMessage(): string {
-    $message = __("Your subscription is currently [link]pending approval[/link]. You’ll soon be able to send once our team reviews your account. In the meantime, you can send previews to your authorized emails.", 'mailpoet');
-    $message = Helpers::replaceLinkTags(
-      $message,
-      'https://kb.mailpoet.com/article/350-pending-approval-subscription',
-      [
-        'target' => '_blank',
-        'rel' => 'noopener noreferrer',
-      ]
-    );
-
-    return "{$message}<br/>";
-  }
-
   /**
    * Returns error $message and $operation for API::RESPONSE_CODE_CAN_NOT_SEND
    */
   private function getCanNotSendError(array $result, $sender): array {
     if ($result['error'] === API::ERROR_MESSAGE_PENDING_APPROVAL) {
       $operation = MailerError::OPERATION_PENDING_APPROVAL;
-      $message = $this->getPendingApprovalMessage();
+      $message = $this->pendingApprovalNotice->getPendingApprovalMessage();
```

Comment:
Nitpick: Not sure if this makes any difference but the original message from `getPendingApprovalMessage()` had a `<br/>` appended at the end

Reply:
Good catch! I'm not sure myself how important it is, but we may as well keep it consistent. I added it back in 1c8cc23bd795a23cda272011b19fda574e32c64d.